### ISSUE TITLE
Greek language stemmer?

### ISSUE THREAD

**Issue by MaghSamana** [wamania/php-stemmer#14]
Hello,
this isn't an issue, but a request. There is a stemming algorithm for greek language.
https://snowballstem.org/algorithms/greek/stemmer.html

Can it be added here?

**Comment by HLeithner** [comment]
@msaari you wrote in issue #21 that you may implement other languages if a snowball algorithm exists. Can I motivate you to keep the Greek stemmer a chance?

**Comment by msaari** [comment]
Yes. There’s an obscene amount of steps in it and the additional complication of using Greek alphabet, but I can look at this. Not before my vacation’s over, so at some point in August or September, depending on how much development time Relevanssi’s next version takes.

**Comment by MaghSamana** [comment]
I can be of help in anything regarding Greek language, alphabet, characters etc, - just message if there's anything needed.

**Comment by msaari** [comment]
I've started work on the Greek stemmer in the `greek` branch. It seems complicated, but we'll see how it turns out once I get to the actual stemming steps.

**Comment by tarasiadis** [comment]
I’m interesting for the addition of Greek language too. If your need any help on Greek alphabet let me know.

**Comment by msaari** [comment]
I'm stuck with this. The Greek alphabet is not the problem; understanding the stemmer is. If someone can help me understand the stemmer details, that'd be helpful.

**Comment by tarasiadis** [comment]
I'm trying to understand the algorithm too, but I'm not the specialist on this! If someone can help it will be very usefull to include Greek language on stemmer.

**Comment by tarasiadis** [comment]
I have found some code for greek stemmer algorithm to help you. I attach it here.
https://u.pcloud.link/publink/show?code=XZSAo7VZBgQQ9HSGre03ltv3ll2wuzaTlBby

**Comment by tarasiadis** [comment]
Is there any progress on this with greek language?

**Comment by msaari** [comment]
No, as I said, I'm stuck and won't be able to progress this further. Hopefully, someone else can pick this up.
